pkgs/servers/pleroma/default.nix

```diff
@@ -18,21 +18,43 @@
 
 beamPackages.mixRelease rec {
   pname = "pleroma";
-  version = "2.7.0";
+  version = "2.8.0";
 
   src = fetchFromGitLab {
     domain = "git.pleroma.social";
     owner = "pleroma";
     repo = "pleroma";
     rev = "v${version}";
-    sha256 = "sha256-2uKVwjxMLC8jyZWW+ltBRNtOR7RaAb8SPO1iV2wyROc=";
+    sha256 = "sha256-8xhBItGl2BvpYB1N1hZbO1xbtYFldUPCiOI4Kvvywaw=";
   };
 
   patches = [ ./Revert-Config-Restrict-permissions-of-OTP-config.patch ];
 
   mixNixDeps = import ./mix.nix {
     inherit beamPackages lib;
     overrides = final: prev: {
+      phoenix = beamPackages.buildMix {
+        name = "phoenix";
+        version = "v1.7.14-websocket-headers";
+        src = fetchFromGitHub {
+          owner = "feld";
+          repo = "phoenix";
+          rev = "v1.7.14-websocket-headers";
```

Comment:
```suggestion
          tag = "v1.7.14-websocket-headers";
```

Reply:
Nice, I did not know about this one. Cool recent addition!

Reply:
In fact this reveals that the "tag" is really a branch. (It will fail if you don't have it cached.)

So it would be better to add the relevant commit to patches. Or specify the full commit as rev.

Reply:
Hmm.

Updated the PR to rebuild this branch applying the patch on top of upstream phoenix.

PTAL